File: specification/specs2/x_pack/machine_learning/start_datafeed/StartDatafeedRequest.ts
@rest_spec_name("xpack.ml.start_datafeed")
class StartDatafeedRequest extends RequestBase {
	timeout: Time;
	@prop_serializer("EpochMillisecondsDateTimeJsonConverter")
	start: Date;
	@prop_serializer("EpochMillisecondsDateTimeJsonConverter")
	end: Date;
}

File: output/csharp/XPack/MachineLearning/StartDatafeed/StartDatafeedRequest.cs
using Nest.Internal;
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class StartDatafeedRequest : RequestBase {
		
		[DataMember(Name="end")]
		public DateTimeOffset End { get; set; }


		[DataMember(Name="start")]
		public DateTimeOffset Start { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }

	}
}
